bittensor/axon.py

```diff
@@ -39,6 +40,9 @@
 from starlette.middleware.base import BaseHTTPMiddleware, RequestResponseEndpoint
 from typing import Dict, Optional, Tuple, Union, List, Callable
 
+# Force asyncio to use the uvloop created event loop. (Much faster)
+asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
```

Comment:
Should only need this once in the program. Try in the main() ?

Reply:
Fixed. Removed from dendrite.py also and put in axon::__init__